pkg/cri/store/image/image.go

```diff
@@ -134,17 +141,38 @@ func getImage(ctx context.Context, i containerd.Image) (*Image, error) {
 
 	id := desc.Digest.String()
 
-	spec, err := i.Spec(ctx)
+	rb, err := content.ReadBlob(ctx, i.ContentStore(), desc)
```

Comment:
A more descriptive name to signify what the blob we just read is

Reply:
Does contentStoreImgBlob sound ok?